packages/components/src/components/hds/accordion/item/index.ts

```diff
@@ -11,10 +11,16 @@ export interface HdsAccordionItemSignature {
     ariaLabel?: string;
     containsInteractive?: boolean;
     isOpen?: boolean;
+    forceState?: 'open' | 'close';
   };
   Blocks: {
-    content?: [];
     toggle?: [];
+    content: [
+      {
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        close: (...args: any[]) => void;
```

Comment:
Could this be `unknown` instead of `any`?

Reply:
Oh interesting (`unknown` vs. `any`). Was looking at this: https://devblogs.microsoft.com/typescript/announcing-typescript-3-0-rc-2/#the-unknown-type

Seems like a better choice if it works for us.

Reply:
`unknown` might be a better fit although, for consistency with other functions, I would be tempted to leave it as `any` and review them holistically.